ansible/roles/gcc/tasks/main.yml

```diff
@@ -0,0 +1,233 @@
+---
+
+#
+# installs gcc 4.9 @ /data/gcc-4.9
```

Comment:
If this file is only run on linuxone-rhel72 then it would be good to say that in a comment here, as it looks like the generic "install gcc" task.

Reply:
will add that comment